romochka_bembot.js

```diff
@@ -56,11 +59,12 @@ const generateBemhtmlLink = bemhtml => {
  * @returns {String} Link
  */
 const generateBemjsonLink = bemjson => {
-	const trimedBemjson = _.trimEnd(bemjson, ';');
+	assert(typeof bemjson === 'string', 'Bemjson must be a string');
+
+	const trimedBemjson = bemjson.replace(/;+$/, '');
```

Comment:
I don't think that use a regexp in that case is a good idea as well as solution on native functions. More straight forward is to use lodash.
Native `startswith()` - yes, why not?
`assert` seems excess because we talk about `onText` event.
Would you change PR to `startswith()` only? Are we convince you?)